scripts/tasks/task-sysex-console.py

```diff
@@ -0,0 +1,141 @@
+#! /usr/bin/env python3
+from datetime import datetime
+from iterfzf import iterfzf
+from pathlib import Path
+import argparse
+import mido
+import os
+import subprocess
+import sys
+
+DEBUG_SYSEX_ID = 0x7D
+DELUGE_PORT_DEFAULT_IN = "Deluge IN"
+DELUGE_PORT_DEFAULT_OUT = "Deluge OUT"
+
+# b.mtime
+# gcc -shared -o scripts/util/pack.so -fPIC src/deluge/util/pack.c
+
+
+def argparser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(
+        prog="sysex-console",
+        description="Listens for deluge sysex and prints console messages"
+    )
+    parser.group = "Development"
+    parser.add_argument(
+        "-I", "--input", help="The device we are reading from", default=DELUGE_PORT_DEFAULT_IN
+    )
+    parser.add_argument(
+        "-O", "--output", help="The device we notify we'll be reading from so they know to read to us", default=DELUGE_PORT_DEFAULT_OUT
+    )
+
+    return parser
+
+def is_verbose_mode() -> bool:
+    return os.environ.get('SYSEX_CONSOLE_VERBOSE', False) or os.environ.get('DBT_VERBOSE', False)
+
+def run_command(command: list, verbose: bool) -> None:
+    if verbose:
+        print(f"Executing command: {' '.join(command)}")
+
+    result = subprocess.run(command, capture_output=True, text=True)
+    
+    if verbose:
+        print(f"Exit code: {result.returncode}")
+        print(f"Output: {result.stdout}")
+        if result.returncode != 0:
+            print(f"Error: {result.stderr}")
+
+    result.check_returncode()
```

Comment:
Since this logic is shared with `task-loadfw`, maybe it could be moved to `util.py` instead?